spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -371,13 +374,20 @@ def get_reviews(app_id, platform, storefront, version_id)
 
         r = request(:get, rating_url)
         all_reviews.concat(parse_response(r, 'data')['reviews'])
+        last_review_date = Time.at(all_reviews[-1]['value']['lastModified']/1000)
+        
+        break if last_review_date < upto_date && !upto_date.nil?
```

Comment:
I would probably make this break and filter here something like this...
```rb
if upto_date && last_review_date < upto_date
  all_reviews = all_reviews.select { |review| Time.at(review['value']['lastModified']/1000) > upto_date  }
  break
end
```

And then remove the multiple returns at the bottom

Reply:
haha cool. I originally had something similar to this, but then I thought I should keep it a one-liner (looked at a couple of other `break` statements). Changed it to like you suggested!
https://github.com/fastlane/fastlane/pull/13040/commits/7c572e21ba96a57a85b0ff75cc0c8853aaaaa48b